addons/web/static/src/views/list/list_renderer.js

```diff
@@ -236,21 +236,54 @@ export class ListRenderer extends Component {
-                            activeField.relatedPropertyField &&
-                            activeField.relatedPropertyField.fieldName === column.name
+                        (field) =>
+                            field.relatedPropertyField &&
+                            field.relatedPropertyField.fieldName === column.name
                     )
-                    .map((activeField) => ({
-                        ...activeField,
-                        id: `${column.id}_${activeField.name}`,
-                        classNames: column.classNames,
-                        optional: "hide",
-                        type: "field",
-                        hasLabel: true,
-                        label: activeField.string,
-                    }));
+                    .map((propertyField) => {
+                        let widget = propertyField.type;
+                        if (["many2one", "many2many"].includes(propertyField.type)) {
+                            if (["res.users", "res.partner"].includes(propertyField.comodel)) {
+                                widget =
+                                    propertyField.type === "many2one"
+                                        ? "many2one_avatar"
+                                        : "many2many_tags_avatar";
+                            } else {
+                                widget =
+                                    propertyField.type === "many2one" ? widget : "many2many_tags";
+                            }
+                        } else if (widget === "tags") {
+                            widget = `property_tags`;
+                        }
+                        const field = getFieldFromRegistry(propertyField.type, widget);
+                        let { relatedFields } = field;
+                        if (relatedFields) {
+                            if (relatedFields instanceof Function) {
+                                relatedFields = relatedFields({ options: {}, attrs: {} });
+                            }
+                            relatedFields = Object.fromEntries(
+                                relatedFields.map((f) => [f.name, f])
+                            );
+                        }
+
+                        return {
+                            ...propertyField,
+                            id: `${column.id}_${propertyField.name}`,
+                            classNames: column.classNames,
+                            optional: "hide",
+                            type: "field",
+                            hasLabel: true,
+                            label: propertyField.string,
+                            sortable: false,
+                            options: {}, // maybe use the options of the properties field ?
+                            modifiers: {}, // maybe use the options of the properties field ?
+                            attrs: {},
+                            field,
+                            widget,
```

Comment:
we need a function somewhere that generates  the fieldInfo (same thing as returned by parseFieldNode) for properties. And here, you should only add list specific keys (optional, type, hasLabel...)